www/main_test.ts

```diff
@@ -2,16 +2,18 @@ import { assertEquals } from "$std/testing/asserts.ts";
 import { TextLineStream } from "$std/streams/delimiter.ts";
 
 Deno.test("CORS should not set on GET /fresh-badge.svg", {
+  sanitizeOps: false,
```

Comment:
is there any way to avoid using `sanitizeOps: false`?

Reply:
@hashrock i have pushed a change. PTAL thanks.